extension/flow/FlowGraph.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let instance = null;
+
+const log = require('../../net2/logger.js')(__filename);
+const rclient = require('../../util/redis_manager.js').getRedisClient();
+
+class FlowGraph {
```

Comment:
change the name here to avoid confusion with class FlowGraph @ FlowManager.js
some class descriptions will be nice